.nais/naiserator-dev.yaml

```diff
@@ -60,6 +58,14 @@ spec:
         - application: syfo-tilgangskontroll
           namespace: teamsykefravr
           cluster: dev-fss
+    outbound:
+      external:
+        - host: "isproxy.dev-fss-pub.nais.io"
+        - host: "pdl-api.dev-fss-pub.nais.io"
+        - host: "skjermede-personer-pip.dev.intern.nav.no"
```

Comment:
Har de laget åpning for syfobehandlendeenhet fra gcp?

Reply:
Jepp